app/models/user.rb

```diff
@@ -7,4 +7,29 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  def average_score
+    ratings = Rating.where(ratee: self)
+    if !ratings.empty?
+      sum = 0
+      ratings.each do |rating|
+        sum += rating.score
+      end
+      average = (sum/ratings.count.to_f).round(2)
```

Comment:
Useless assignment to variable - `average`.<br>Surrounding space missing for operator `/`.